src/SFA.DAS.ApprenticeCommitments.Jobs.Functions/Infrastructure/Startup.cs

```diff
@@ -19,7 +21,18 @@ public override void ConfigureAppConfiguration(IFunctionsConfigurationBuilder bu
         public override void Configure(IFunctionsHostBuilder builder)
         {
             builder.ConfigureLogging();
-            builder.ConfigureNServiceBus();
+
+            builder.UseNServiceBus(() =>
+            {
+                var configuration = new ServiceBusTriggeredEndpointConfiguration(
+                    endpointName: QueueNames.ApprenticeshipCreatedEvent,
+                    connectionStringName: "AzureWebJobsServiceBus");
```

Comment:
```suggestion
                    connectionStringName: "NServiceBusConnectionString");
```
Using the pre-existing NServiceBusConnectionString app setting